contentcuration/contentcuration/frontend/shared/views/policies/PrivacyPolicyModal.vue

```diff
@@ -1,26 +1,276 @@
 <template>
 
-  <PoliciesModal
-    v-model="dialog"
-    :requirePolicyAcceptance="requirePolicyAcceptance"
-    :policy="policyName"
-  >
-    <p>{{ $tr('paragraph1') }}</p>
-    <p>{{ $tr('paragraph2') }}</p>
-    <p>{{ $tr('paragraph3') }}</p>
-    <p>
-      <ActionLink
-        :text="$tr('linkText')"
-        target="_blank"
-        href="https://learningequality.org/about/values/"
-      />
-    </p>
-    <p>{{ $tr('paragraph4') }}</p>
+  <PoliciesModal :policy="policyName">
+    <div class="tos-wrapper">
+      <p class="emphasis">
+        {{ $tr('introductionHeader') }}
```

Comment:
We'll need to unwrap these strings as we don't want these to be picked up for translation. Could you copy/paste the corresponding text from $trs here instead? 

Also, since we're not translating the strings, you'll need to add the `dir=auto` to the <p> tag to automatically align the text according to the language it's in

Reply:
This will resolve the issue described here https://github.com/learningequality/studio/issues/2631#issuecomment-740238625

Reply:
@jayoshih Could you update the issue for updating the policy? These details were not mentioned